package.json

```diff
@@ -67,7 +67,7 @@
     "react": "0.13.x || 0.14.x"
   },
   "optionalDependencies": {
-    "jsdom": "^6.1.0",
+    "jsdom": "~3.1.2",
```

Comment:
Please use `^`, instead of `~` - older nodes are fine, but older npms don't even work properly with the registry, so there's no reason to support them.